regression-test/suites/load_p0/stream_load/test_stream_load_2pc.groovy

```diff
@@ -317,9 +317,9 @@ suite("test_stream_load_2pc", "p0") {
         '',
         """
         "dynamic_partition.enable" = "true",
-        "dynamic_partition.time_unit" = "MONTH",
-        "dynamic_partition.start" = "-3",
```

Comment:
There is no need to change this, as it will result in longer case execution time.